rust/operator-binary/src/superset_controller.rs

```diff
@@ -208,15 +217,29 @@ fn build_node_rolegroup_service(
                 &rolegroup.role,
                 &rolegroup.role_group,
             )
+            .with_label(
+                "statsd-exporter",
+                statsd_exporter_version(superset).context(NoStatsdExporterVersionSnafu)?,
+            )
+            .with_label("prometheus.io/scrape", "true")
+            .with_label("prometheus.io/port", METRICS_PORT.to_string())
```

Comment:
I like this but the port label is not required. The scraping jobs will check for `metrics` container ports. We can leave it in though.

Reply:
removed as discussed